src/components/map-page-components/map-view/MapView.tsx

```diff
@@ -32,6 +32,24 @@ const MapView: React.FC = () => {
   });
 
   const mapId = '76c08a2450c223d9';
+  var markersArray: google.maps.Marker[] = [];
+
+  interface KeyValuePair {
+    key: string;
+    value: string;
+}
+
+  //This KeyValuePair array that stores names to be shortHand-ed. 
+  //To add a new area, wrap a key value pair with {}.
+  let shortHandNames: KeyValuePair[] = [
+  {key: "North End", value: "N.E"}, 
```

Comment:
Can you change the shorthand value to not use the ".". So just NE, WE, LD, BH, etc.